src/locales/en/ui.yml

```diff
@@ -223,6 +238,17 @@ views:
     manageFavorites: Manage Favorites
   editor:
     needlogin: You'll need to login to make edits to Holodex video and song metadata.
+    changetopic:
+      Contents:
```

Comment:
maybe call it changetopic.info and changetopic.button, kind of nit picking since i18n ally would probably handle it, but makes it slighlty more readable than changtopic[0]